packages/graph/hash_graph/tests/integration/postgres/property_type.rs

```diff
@@ -0,0 +1,56 @@
+use crate::postgres::DatabaseTestWrapper;
+
+#[test]
+fn insert() {
+    let age_pt = serde_json::from_str(crate::test_data::property_type::AGE_V1)
+        .expect("could not parse property type");
+
+    let mut database = DatabaseTestWrapper::new();
+    database
+        .create_property_type(age_pt)
+        .expect("could not create property type");
+}
+
+#[test]
+fn query() {
+    let interests_pt = serde_json::from_str(crate::test_data::property_type::INTERESTS_V1)
```

Comment:
Do we want to insert the necessary data types here? I know we don't currently validate for them but we might want to write the tests as if we did. I don't mind